feg/gateway/policydb/flow_parser.go

```diff
@@ -150,18 +154,24 @@ func parseAddress(addr string) (*address, error) {
 		return nil, fmt.Errorf("Invalid format for address %s", addr)
 	}
 	ipAddr := matches[0]
+	version := protos.IPAddress_IPV4
 	if ipAddr == "any" {
-		return &address{ip: "", port: 0}, nil
+		return &address{ip: "", version: version, port: 0}, nil
+	}
+
+	// Easy check if its an ipv6 addr
+	if strings.Count(ipAddr, ":") >= 2 {
+		version = protos.IPAddress_IPV6
```

Comment:
Maybe you don't need it here, but if you really want to check ans parse ip you con use module `net`

```
import net
...

if ipv6 := net.ParseIP(request.IPv6Addr); ipv6 != nil {

```